A.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="A.css">
+</head>
+<body>
+
```

Comment:
- [ ] [REQUIRED]  Great job on giving this pull request a descriptive title. However, your Pull Request should have a descriptive summary describing introduced changes in more detail. Take note of this as it is best practice to have your PR summary describe the things you implemented in the project. 
e.g Title -> Hello Microverse project and the summary -> Mention the things you did in brief detail.
See the image below for an example of a descriptive PR summary 👇

![](https://user-images.githubusercontent.com/93039359/191243176-550fc938-97e4-4cf7-969c-ce112fd46536.png)